wg-notes/annotations-ucr/index.html

```diff
@@ -1,131 +1,198 @@
+        <section>
+            <h2>Annotating</h2>
+            <section>
+                <h3>Annotating a textual section</h3>
+                <p>
+                    A user decides to annotate a textual section of an EPUB. She
+                    selects the section, triggers the annotation affordance,
+                    optionally enters a note, selects a highlight mode and
+                    color. She then saves the annotation. The selected section
+                    appears on the page with the chosen highlight.
+                </p>
+            </section>
+            <section>
+                <h3>Annotating an image</h3>
+                <p>
+                    A user decides to annotate an image of an EPUB. He selects
+                    the image and triggers the annotation affordance. The
+                    annotation feature is then identical to the one associated
+                    with a textual selection.
+                </p>
+            </section>
+            <section>
+                <h3>Annotation parts of an image</h3>
+                <p>
+                    A user consults an EPUB publication that includes detailed
+                    diagrams (e.g., in SVG) and images (e.g., in JPG or PNG)
+                    visualizing a research data set. She analyses the data by
+                    annotating <em>parts</em> of the diagrams and/or images by
+                    selecting rectangular areas of the data visualization and
+                    shares these annotations with her colleagues.
+                </p>
+            </section>
+        </section>
+        <section>
+            <h2>Formatting annotations</h2>
+            <p>
+                A user adds an annotation to an EPUB. He adds some formatting to
+                the text of the annotation: emphasize, italic, underline,
+                subtitles.
+            </p>
+        </section>
+        <section>
+            <h2>Bookmarking</h2>
+            <p>
+                A user decides to bookmark a location in a reflowable EPUB. The
+                cursor is located where the user clicks on the screen, or at a
+                default location (often the top-left corner of the screen, for
+                left-to-right content). The user triggers the bookmark
+                affordance, optionally selects a highlight mode & color, and
+                saves the bookmark. A bookmark icon appears on the page, near
+                the line where the cursor was positioned when the bookmark was
+                created.
+            </p>
+        </section>
+        <section>
+            <h2>Exporting &amp; importing annotations and bookmarks</h2>
+            <section>
+				<h3>Publishing a book with annotations</h3>
+            	<p>
+	                A famous author annotates a book she has just written, and deals
+	                with his publisher a specific edition of the EPUB version of the
```

Comment:
Consistency within paragraph:

```suggestion
	                with her publisher a specific edition of the EPUB version of the
```

Reply:
good. as this PR has been merge, I'll add it to the next evolution of the text.